lua/apisix/plugins/proxy-rewrite.lua

```diff
@@ -70,6 +70,13 @@ function _M.rewrite(conf, ctx)
         end
     end
 
+    local upstream_uri = conf.uri or ctx.var.uri
+    if ctx.var.is_args == "?" or str_sub(ctx.var.request_uri, -1) == "?" then
```

Comment:
Is  there a test case for `str_sub(ctx.var.request_uri, -1) == "?"` ? I am not get the point.

Reply:
I want to get the trailing `?` symbol in this way, but this way is wrong and has been deleted.